### ISSUE TITLE
Conference Issues Closure Automation

### ISSUE THREAD

**Issue by JohananOppongAmoateng** [BlackPythonDevs/blackpythondevs.github.io#523]
Are the conference issues automatically closed after the end date? If not I am proposing we implement that in the GitHub workflow.

**Comment by kjaymiller** [comment]
> Are the conference issues automatically closed after the end date? If not I am proposing we implement that in the GitHub workflow.

@JohananOppongAmoateng - No, the issues should close once the issue is approved (which is currently not working).

This is broken and a new issue should be created.

There was an update by me yesterday to work on this.

**Comment by JohananOppongAmoateng** [comment]
@kjaymiller is the bug you are talking about fixed

**Comment by kjaymiller** [comment]
No. I was saying we shoudl create a new issue to fix that bug. GitHub actions are one of the things that maybe we should let admins work on because testing is incredibly hard. Although you can test by adding GH Actions and testing locally

**Comment by JohananOppongAmoateng** [comment]
Okay 👍 noted
